composer.json

```diff
@@ -3,8 +3,8 @@
     "type": "library",
     "license": "proprietary",
     "require": {
-        "php": "^7.4 || ^8.0",
-        "dvsa/mot-cpms-queues": "^1.0.1",
+        "php": "^8.2",
+        "dvsa/mot-cpms-queues": "dev-feature-bl-17033-php-82-update",
```

Comment:
Create a tag in dependency repos and use it here. You can create a tag before merging to master, and after merging you can correct it if needed.